services/index.ts

```diff
@@ -33,6 +34,34 @@ const validateRequest = (req, res, next) => {
     next()
 }
 
-app.use('/graphql', bodyParser.json(), validateRequest, graphqlExpress({ schema }))
+app.use(
+    '/graphql',
+    cookieParser(),
+    bodyParser.json(),
+    validateRequest,
+    graphqlExpress((req, res) => ({
+        schema,
+        context: ({ req, res }) => ({
+            req,
+            res,
+            userPayload: userPayloadFromRequest({ req, res }),
```

Comment:
should be named `token` could be anonymous function or create it outside of use(should be 1 line).